scripts/shared/execute.ts

```diff
@@ -5,13 +5,13 @@ import {infoLog} from 'ng-morph';
 export function execute(shell: string, options?: Partial<CommonExecOptions>): string {
     infoLog(`ᐅ ${shell}`);
 
-    return execSync(
+    const std = execSync(
         shell,
         options ?? {
             stdio: 'inherit',
             encoding: 'utf8',
         },
-    )
-        ?.toString()
-        .trim();
+    ) as Buffer | undefined;
```

Comment:
How does it return `undefined`? Types say it returns `Buffer | string`. Instead of `.toString()` if you are worried maybe it can just be `String(...)`?

Reply:
fixed